### PR TITLE
attitude: implement velcompass functionality

### PR DESCRIPTION
**Pull request by mlyle**
Uses GPS to determine direction of travel, for fixed wing and other
craft that point the way they're going.

### CODE REVIEW

File: shared/uavobjectdefinition/stateestimation.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
 			</description>
       <options>
         <option>Complementary</option>
+        <option>Complementary+VelCompass</option>
```
Comment (tracernz) [inline shared/uavobjectdefinition/stateestimation.xml:14]: Please add brief description to field description.

Comment (mlyle) [reply shared/uavobjectdefinition/stateestimation.xml:14]: Done, thx.

Comment (tracernz) [reply shared/uavobjectdefinition/stateestimation.xml:14]: Great 👍

**Comment by tracernz** [comment]
Hmmm
```
+ git merge-base origin/next HEAD
fatal: Not a valid object name origin/next
```

**Review by tracernz — APPROVED** [review]

**Comment by tracernz** [comment]
test this please

**Comment by tracernz** [comment]
I believe there have been positive flight test reports on this (IRC)?

**Comment by mlyle** [comment]
The 8 seconds of flight the plane survived, it seemed to be doing the right thing.

**Comment by tracernz** [comment]
8 seconds ought to be enough for anyone right? 😆

**Pull request merged**